types/aws-lambda/index.d.ts

```diff
@@ -181,6 +188,45 @@ interface ProxyResult {
 }
 
 /**
+ * API Gateway CustomAuthorizer AuthResponse.
+ * http://docs.aws.amazon.com/apigateway/latest/developerguide/use-custom-authorizer.html#api-gateway-custom-authorizer-output
+ */
+interface AuthResponse {
+    principalId: string;
+    policyDocument: PolicyDocument;
+    context?: AuthResponseContext;
+}
+
+/**
+ * API Gateway CustomAuthorizer AuthResponse.PolicyDocument.
+ * http://docs.aws.amazon.com/apigateway/latest/developerguide/use-custom-authorizer.html#api-gateway-custom-authorizer-output
+ */
+interface PolicyDocument {
+    Version: string;
+    Statement: [Statement];
+}
+
+/**
+ * API Gateway CustomAuthorizer AuthResponse.PolicyDocument.Statement.
+ * http://docs.aws.amazon.com/apigateway/latest/developerguide/use-custom-authorizer.html#api-gateway-custom-authorizer-output
+ */
+interface Statement {
+    Action: string;
+    Effect: string;
+    Resource: string | [string];
```

Comment:
I was taught the following documents from AWS engineers.
http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html

We can know that "Resources" is available in the array in this document.

```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Permission",
      "Action": [
        "execute-api:Execution-operation"           
      ],
      "Resource": [
        "arn:aws:execute-api:region:account-id:api-id/stage/METHOD_HTTP_VERB/Resource-path"
      ]
    }
  ]
}
```